eth/protocols/diff/handler_test.go

```diff
@@ -134,9 +134,9 @@ func testGetDiffLayers(t *testing.T, protocol uint) {
 	missDiffPackets := make([]FullDiffLayersPacket, 0)
 
 	for i := 0; i < 100; i++ {
-		number := uint64(rand.Int63n(1024))
-		if number == 0 {
-			continue
+		// Find a non 0 random number
+		var number uint64
+		for ; number == 0; number = uint64(rand.Int63n(1024)) {
```

Comment:
maybe use `number := uint64(rand.Int63n(1023) + 1)` ? `rand.Int63n` range is [0,n)

Reply:
The purpose is to find out a non zero number, so as to try `1023/1024` possibility.
But another problem is that we should do some re-seed to avoid pseudo randomness.

Reply:
use `crypto/rand`, don't need re-seed